cli/lib/version.rs

```diff
@@ -36,12 +36,16 @@ pub static DENO_VERSION_INFO: std::sync::LazyLock<DenoVersionInfo> =
         }
       });
 
-    DenoVersionInfo {
-      deno: if release_channel == ReleaseChannel::Canary {
+    let deno = libsui::find_section("denoversion")
```

Comment:
I don't understand why libsui is being introduced here. Why does the previous logic not work? Add a comment